### PR TITLE
feat(l1): enhance el assertoor ci test by cherrypicking more checks

### LINKED ISSUE #953: [CI] Cherry-pick assertoor-test checks that could be added to the CI run
The idea is to add additional useful el checks outside of the stability ones from the [`assertoor-test`](https://github.com/ethpandaops/assertoor-test/tree/master/assertoor-tests) ones that are able to run in the CI without affecting the total time.

### PR DESCRIPTION
**Pull request by rodrigo-o**
**Motivation**

This PR tries to add as much test as possible to our assertoor CI run.

**Description**

The idea is to add  every check that is useful for validating the el execution without incurring in long running times to avoid making the CI slower. Right now every useful check from the [supported](https://github.com/ethpandaops/assertoor/wiki#supported-tasks-in-assertoor) is added except for:
- ✅ `check_consensus_finality` | time: ~10m
- ✅ `check_consensus_attestation_stats` | time: ~10m

Apart from checks, we are not using most of the `generate_*` tasks due to timing or errors:
- ✅ `run_transaction_test` |  time: ~10m
- ✅ `run_blob_transaction_test` | time: ~6m
- ❎  `run_opcodes_transaction_test` | time: ~5m (We have issues processing some of the transactions)

For the already available assertor playbooks, [see here](https://github.com/ethpandaops/assertoor-test/tree/master/assertoor-tests).

The `run_blob_transaction_test` is the only one added until now from transaction checks and the total test take nearly 15m (which is less than the largest hive test in the CI).

Closes #953

### CODE REVIEW

File: test_data/network_params.yaml [hunk 1]
```diff
@@ -17,5 +20,5 @@ assertoor_params:
   run_stability_check: false
   run_block_proposal_check: false
   tests:
-    - 'https://raw.githubusercontent.com/lambdaclass/lambda_ethereum_rust/refs/heads/assertoor-run-on-cd/test_data/el-stability-check.yml'
+    - 'https://raw.githubusercontent.com/lambdaclass/lambda_ethereum_rust/refs/heads/enhance-el-assertoor-ci-test/test_data/el-stability-check.yml'
```
Comment (rodrigo-o) [inline test_data/network_params.yaml:23]: After the reviews this will be changed to point to main instead.

File: test_data/network_params.yaml [hunk 2]
```diff
@@ -1,21 +1,23 @@
 participants:
   - el_type: geth
     cl_type: lighthouse
-    count: 2
+    validator_count: 32
+  - el_type: geth
+    cl_type: prysm
+    validator_count: 32
```
Comment (mpaulucci) [inline test_data/network_params.yaml:7]: why this change?

Comment (rodrigo-o) [reply test_data/network_params.yaml:7]: I wanted to have at least a different CL client for variety (to make sure other CLs receive and accept our blocks). Do you prefer to add other CL client/s instead of this one?

Regarding the validator count, default is 64, I lowered it to 32 to speed up the block proposal check (because I thought it was a per-validator thing due to variable naming in the definition of the check when in fact it was a per-client one). I could revert the validator count change if preferred and check times, I don't think it'll impact.

Comment (mpaulucci) [reply test_data/network_params.yaml:7]: Im fine with more CL clients but I thought it would affect the job time (more clients, more compilation right?). Anyway, not a big deal since it seems we're good time wise

**Comment by mpaulucci** [comment]
Are all blocks proposed by our execution client empty blocks (i.e no transactions)?

File: test_data/network_params.yaml (hunk 2 above)
Comment (rodrigo-o) [reply test_data/network_params.yaml:7]: It will affect kurtosis setup for sure when starting the client (not by a lot as far as I could see) and yes probably also affect some of the checks, but fortunately it wasn't so big of an impact.

**Comment by rodrigo-o** [comment]
> Are all blocks proposed by our execution client empty blocks (i.e no transactions)?

In most of the test, yes, our blocks are empty. But, due to the addition of the `run_blob_transaction_tests` during that execution our blocks have blobs (the test generates transactions for all clients instead of spamming one and expecting P2P to be in place). Unfortunately the `run_transaction_tests` which adds legacy and dfyne transactions and checks it's addition makes the job to last past the longest run (19m), but it worked, I already tested it!

**Comment by mpaulucci** [comment]
But if our blocks have blobs, they have a associated transaction too, right?

**Comment by rodrigo-o** [comment]
> But if our blocks have blobs, they have a associated transaction too, right?

Yes, exactly. But just to be clear that happens only while that specific test is running, for most of the rest of the run our blocks are empty.

**Comment by mpaulucci** [comment]
Ok, but at least we have one test with transactions proposed by us!

**Review by mpaulucci — APPROVED** [review]

**Comment by rodrigo-o** [comment]
> Ok, but at least we have one test with transactions proposed by us!

Yes! and here is a proof of that:

image

File: test_data/network_params.yaml (hunk 1 above)
Comment (rodrigo-o) [reply test_data/network_params.yaml]: Done
